modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -640,6 +640,51 @@
 		<td>//div[contains(@class,'card-body')]//button[contains(text(), '${key_optionDefaulValue}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ADD_OBJECT_FOLDER</td>
+		<td>//button[contains(@class, 'component-action btn') and contains(@aria-label, 'Add Object Folder')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>CREATE_FOLDER_BUTTON</td>
+		<td>//button[contains(text(),'Create Folder')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>EDIT_LABEL_AND_ERC</td>
+		<td>//button[contains(@value, 'editFolder')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>FOLDER_ACTIONS</td>
+		<td>//button[contains(@aria-label, 'folder-actions')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>EDIT_FOLDER_PERMISSIONS</td>
+		<td>//button[contains(@value, 'folderPermissions')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>DELETE_FOLDER</td>
+		<td>//button[contains(@value, 'deleteFolder')]</td>
+		<td></td>
+	</tr>
```

Comment:
If the element structure is similar, you can just create a generic path and pass in context using a var. For example:
`//button[contains(@value, '${key_value}')]`

That way you can just call the button and pass in `deleteFolder` or `folderPermissions`, it let's us create one path instead of creating multiple, similar ones.

I'm not going to enforce this for now but keep that in mind for the future. We can always revisit this and refactor the paths.